test/utils/noderesourcetopologies/noderesourcetopologies.go

```diff
@@ -256,6 +256,15 @@ func ZoneResourcesMatchesRequest(resources []nrtv1alpha1.ResourceInfo, requests
 	return true
 }
 
+func FilterByPolicies(list []nrtv1alpha1.NodeResourceTopology, policies []nrtv1alpha1.TopologyManagerPolicy) []nrtv1alpha1.NodeResourceTopology {
+	var filteredNrts []nrtv1alpha1.NodeResourceTopology
+	for _, policy := range policies {
```

Comment:
we can refactor more in the future to iterate over the NRT slice just once. But this can wait.